### PR TITLE
[Jormungandr] Handle providers from db

### PR DESCRIPTION
**Pull request by mbounabi**
https://jira.kisio.org/browse/NAVP-1253

### CODE REVIEW

File: source/jormungandr/jormungandr/equipments/sytral.py [hunk 1]
```diff
@@ -49,9 +49,10 @@ class SytralProvider(object):
 
     def __init__(self, url, **kwargs):
         self.url = url
+        self.timeout = kwargs.get('timeout', app.config['CIRCUIT_BREAKER_SYTRAL_TIMEOUT_S'])
```
Comment (kinnou02) [inline source/jormungandr/jormungandr/equipments/sytral.py]: timeout and circuit breaker timeout isn't the same things

**Review by kinnou02 — COMMENTED** [review]
A bit of tests for the loading from the database would be great

**Comment by kinnou02** [comment]
Still no test on `EquipmentProviderManager` which a `providers_getter`, it's pretty easy to mock it.

**Comment by mbounabi** [comment]
@kinnou02 , Work In Progress, update today!

**Comment by mbounabi** [comment]
Jenkins, test this please!

**Review by kinnou02 — APPROVED** [review]

**Review by nonifier — APPROVED** [review]

**Comment by mbounabi** [comment]
Jenkins, test this please!

**Review by xlqian — APPROVED** [review]

**Pull request merged**